src/context/OnlineStatusContext.js

```diff
@@ -25,8 +49,12 @@ export const OnlineStatusProvider = ({ children }) => {
 		console.log("Online status changed to ", isOnline);
 	}, [isOnline]);
 
+	useEffect(() => {
+		console.log("Connectivity status change to ", connectivityQuality);
+	}, [connectivityQuality]);
+
 	return (
-		<OnlineStatusContext.Provider value={{ isOnline }}>
+		<OnlineStatusContext.Provider value={{ isOnline, connectivityQuality }}>
```

Comment:
I would probably call this `connectionQuality` as it's shorter and easier to read and type, while not losing any meaning. :slightly_smiling_face: